.textlintrc.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  rules: {
+    '@lmc-eu/textlint-rule-preset-lmc': {
+      'title-case': {
+        headingLevels: [6],
+        exclude: [
+          '@lmc-eu',
+          '@lmc-eu/stylelint-config',
+          '@lmc-eu/eslint-config-react',
+          '@lmc-eu/eslint-config-react/optional',
+          '@lmc-eu/eslint-config-base',
+          '@lmc-eu/eslint-config-base/optional',
+          '@lmc-eu/eslint-config-base/legacy',
+          '@lmc-eu/eslint-config-base/whitespace',
+          '@lmc-eu/commitlint-config',
+          '@lmc-eu/textlint-rule-preset-lmc',
```

Comment:
Does it support a wild card?

```suggestion
          '@lmc-eu',
          '@lmc-eu/*',
```

Reply:
No, it does not. https://github.com/sapegin/textlint-rule-title-case/blob/master/index.js#L88 Only equal comparison.